src/kubernetes/config.aat.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${SERVICE_NAME}-config
+  namespace: ${NAMESPACE}
+data:
+  IDAM_API_URL: https://preprod-idamapi.reform.hmcts.net:3511
+  IDAM_AUTHENTICATION_WEB_URL: https://idam.preprod.ccidam.reform.hmcts.net
+  IDAM_S2S_AUTH: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+  PAY_URL: http://payment-api-aat.service.core-compute-aat.internal
+  CLAIM_STORE_URL: http://cmc-claim-store-aat.service.core-compute-aat.internal
```

Comment:
Hi
Please don't link dependant changes together, you should be testing in isolation it will give false confidence on a build and may cause breakages.
We don't deploy together so we shouldn't test together until the change is on master.

I'm not sure if we have formal documentation on this but every team should have been made aware of this as part of onboarding to the pipeline and any sessions where this has came up.
I did some writing on this when another team asked about it.

If more formal documentation is needed I can do it too:
https://github.com/hmcts/cnp-jenkins-library/pull/270#issuecomment-411675136

Reply:
makes sense 👍